meinberlin/apps/mapideas/forms.py

```diff
@@ -16,11 +17,17 @@ def __init__(self, *args, **kwargs):
             polygon=self.settings.polygon)
         self.fields['point'].error_messages['required'] = _(
             'Please locate your proposal on the map.')
+        self.fields['category'].widget = widgets.SelectCategoryWithIconWidget(
+            choices=self.fields['category'].choices,
+            attrs={'qs': self.fields['category'].queryset})
 
     class Meta:
         model = models.MapIdea
         fields = ['name', 'description', 'category', 'point', 'point_label']
 
+    class Media:
+        js = ('js/category_formset.js',)
```

Comment:
we don't need the `category_formset.js` the `select_dropdown.js` is already required in `adhocracy4/index.js`
but we need a small javascript to initialize the select onLoad (similiar but simpler then in `category_formset.js`